plugin/src/main/java/com/google/errorprone/xplat/checker/XplatBans.java

```diff
@@ -0,0 +1,359 @@
+// Copyright 2020 Google LLC
```

Comment:
Should this file be forked from another file?

Reply:
Yes, not sure what happened. The old file was called JodaTimeClassBan.